README.md

```diff
@@ -224,15 +224,15 @@ be placed in a `maintenance_tasks.rb` initializer.
 Exceptions raised while a Task is performing are rescued and information about
 the error is persisted and visible in the UI.
 
-If your application uses Bugsnag to monitor errors, the gem will automatically
-notify Bugsnag of any errors raised while a Task is performing.
-
-If you want to integrate with another exception monitoring service or customize
-error handling, a callback can be defined:
+If you want to perform operations with errors raised during a Task execution,
+such as reporting them to an exception monitoring service like Bugsnag,
+you can define an error handler. Simply create a proc that takes an error as
+argument. Maintenance Tasks will then call the error handler proc whenever an
+error is raised.
```

Comment:
IMO the original text from the revert was clearer, we may be able to improve on it but this is not.

```suggestion
If you want to integrate with an exception monitoring service (e.g. Bugsnag),
you can define an error handler:
```